android/examples/unrooted_android.c

```diff
@@ -270,7 +270,7 @@ static void print_device(libusb_device *dev, libusb_device_handle *handle)
 }
 
 
-// fileDescriptor = is the native File Descriptor obtained in Java and transfered to native over JNA for Example.
+// fileDescriptor = is the native File Descriptor obtained in Java and transferred to native over JNA, for example.
```

Comment:
We should probably use /* */ instead of // for consistency.

Reply:
Done.